kafkaesk/consumer.py

```diff
@@ -169,11 +170,9 @@ async def __call__(self) -> None:
             while not self._close:
                 try:
                     if not self._consumer.assignment():
-                        self._health_metric(False)
-                        logger.info(f"Consumer {self} waiting for assignment...")
-                        await asyncio.sleep(0.5)
-                    else:
-                        await self._consume()
+                        await asyncio.sleep(2)
```

Comment:
why 2?

Reply:
500ms was way too low for the coordinator to assign partitions, with our current config the reassign process is in the vicinity of minutes. Maybe we want to remove this and allow getmany to return an empty list of records if there is no assigned partitions.